.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
A PS-ism I'm not fond of. If you're adamant about non-reproducible builds, make your case

Reply:
Will remove.